x/multi-staking/keeper/msg_server.go

```diff
@@ -206,23 +206,28 @@ func (k msgServer) CancelUnbondingDelegation(goCtx context.Context, msg *staking
 		return nil, fmt.Errorf("not allow coin")
 	}
 
-	unbondEntry, found := k.keeper.GetUnbondingEntryAtCreationHeight(ctx, multiStakerAddr, valAcc, msg.CreationHeight)
-	if !found {
-		return nil, fmt.Errorf("unbondEntry not found")
+	unlockID := types.MultiStakingUnlockID(msg.DelegatorAddress, msg.ValidatorAddress)
+	cancelMSCoin, err := k.keeper.DecreaseUnlockEntryAmount(ctx, unlockID, msg.Amount.Amount, msg.CreationHeight)
```

Comment:
why don't we use `GetUnlockEntryAtCreationHeight` and `SetMultiStakingUnlockEntry`

Reply:
Because I need more information inside cancel unbonding func, I've written some new functions at the latest commit, pls check it out.